src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -829,10 +839,11 @@ private void ProcessListeningThread(object state)
         /// PowerShell Direct Windows Server Container connection and management.
         /// </summary>
         /// <param name="configurationName">Name of the configuration to use.</param>
-        internal static void RunServerMode(string configurationName)
+        /// <param name="debugPipeName">Name of the pipe to use if it was specified at startup.</param>
+        internal static void RunServerMode(string configurationName, string debugPipeName)
```

Comment:
PowerShell Direct should never need to create a custom named IPC pipe, passing in `debugPipeName` is unnecessary.